src/lambdas/serveExpressApp.js

```diff
@@ -26,12 +27,20 @@ const binaryMimeTypes = [
   'text/xml',
 ];
 
-const server = awsServerlessExpress.createServer(app, null, binaryMimeTypes);
-const isProd = typeof process.env.NODE_ENV !== 'undefined' && process.env.NODE_ENV === 'production';
+function isProd() {
+  const envVar = config.env();
+  return typeof envVar !== 'undefined' && envVar === 'production';
+}
 
-export default (event, context) => {
-  if (isProd) {
+let lambdaExpressServer;
+export default async (event, context) => {
+  if (!lambdaExpressServer) {
+    console.log('Creating new Express server');
+    const expressApp = await app();
+    lambdaExpressServer = awsServerlessExpress.createServer(expressApp, null, binaryMimeTypes);
```

Comment:
Let's test this in dev when its up to see the implications (if any) it has for cold starts